src/main.py

```diff
@@ -169,6 +169,12 @@ def pdf_uri(triple):
     filename = "elife-%s-v%s.pdf" % (utils.pad_msid(msid), version) # ll: elife-09560-v1.pdf
     return cdnlink(msid, filename)
 
+def xml_uri(triple):
+    """predict an article's xml url."""
+    content_type, msid, version = triple
```

Comment:
content-type is not needed anymore here

Reply:
You are right. I have now removed it. I felt pressured to keep it due to the name `triple` 😱.